src/events/crawler/scrape-data/normalize-locations.js

```diff
@@ -1,13 +1,11 @@
 import path from 'path';
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
Much better to scope these. Linter would have caught that the country-codes wasn't used if we'd done it in the first place.